upstream-community-operators/kiali/kiali.v0.18.1.clusterserviceversion.yaml

```diff
@@ -206,7 +206,7 @@ spec:
                     fieldRef:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
-                  value: "kiali-operator"
+                  value: "Kiali Operator"
```

Comment:
Not sure this needs to change, this might be causing CI to fail. Undo this change, and only change the display name